presto-hive/src/main/java/com/facebook/presto/hive/parquet/reader/ParquetColumnReader.java

```diff
@@ -200,7 +200,6 @@ private void readNextPage()
     {
         page = pageReader.readPage();
         validateParquet(page != null, "Not enough values to read in column chunk");
-        pageValueCount = page.getValueCount();
```

Comment:
Do we need the `pageValueCount` field at all? I guess in all places that we read this field we have either access to the page itself or we pass the count as an argument, so seems like we can just remove it.